internal/pkg/api/metadata.go

```diff
@@ -0,0 +1,81 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/docker/go-units"
+)
+
+func parseDownloadRate(jsonDownloadRate json.RawMessage) (*float64, error) {
```

Comment:
We should adjust the openapi spec for the download rate to be:
```yaml
oneOf:
  - type: string
  - type: integer
```

Reply:
I am wondering if we should change the openAPISpec, as this mostly a temporary fix to support a client that did not sent the correct value.

Reply:
Don't make the error an officially accepted value. Tolerate it but don't encourage it's use.

Reply:
Yes I will add one with a random string input already had one with `MBps`